cmd/controller-manager/controller-manager.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	bucketcontroller "github.com/kubernetes-sigs/container-object-storage-interface-api/controller"
+	"github.com/kubernetes-sigs/container-object-storage-interface-controller/pkg/bucketrequest"
+
+	"github.com/golang/glog"
+)
+
+var ctx context.Context
+var cmd = &cobra.Command{
+	Use:           "controller-manager",
+	Short:         "central controller for managing bucket* and bucketAccess* API objects",
+	SilenceErrors: true,
+	SilenceUsage:  true,
+	RunE: func(c *cobra.Command, args []string) error {
+		return run(args)
```

Comment:
You can use `Command`s `Context()` and pass it to `run` instead: https://godoc.org/github.com/spf13/cobra#Command.Context

Reply:
Fixed. Using cmd context now.